dev-project/pipelinewise-config/tap_twilio.yml

```diff
@@ -0,0 +1,93 @@
+---
+
+# ------------------------------------------------------------------------------
+# General Properties
+# ------------------------------------------------------------------------------
+id: "twilio"                           # Unique identifier of the tap
+name: "Twilio"                         # Name of the tap
+type: "tap-twilio"                     # !! THIS SHOULD NOT CHANGE !!
+owner: "somebody@foo.com"              # Data owner to contact
```

Comment:
@EricPulido what these values should be?

Reply:
I am guessing the values should remain dummy variables for the PR and then when configuring in production we add the real values, right @koszti ?

Reply:
yes, that's correct